packages/wast-parser/src/fsm.js

```diff
@@ -10,7 +10,8 @@ type TransitionList<Q> = { [Q]: Array<TransitionEdge<Q>> };
 function makeTransition<Q>(
   regex: RegExp,
   nextState: Q,
-  { n = 1, allowedSeparator }: { n: number, allowedSeparator: ?string } = {}
+  // $FlowIgnore
+  { n = 1, allowedSeparator } = {}
```

Comment:
I couldnt get flow to accept this. I tried 
```
{ n = 1, allowedSeparator } : { n: number, allowedSeparator: ?string } = {}
```
without success. (All calls to the function are marked with "incompatible type with object literal").

Reply:
I think the correct type is `{n: number, allowedSperator?: string }` since allowedSperator is optional.

Reply:
Fixed in https://github.com/xtuc/webassemblyjs/pull/215/commits/9b3f05b375dbeac2e80572525d467e604ee96cbb.

`n` and `allowedSeperator` are optional keys, it's `n?: number` in flow. `n: ?number` means that the key is needed but can be undefined/null.